example/README.md

```diff
@@ -3,7 +3,7 @@
 This is an example package for React-Scrollbar-Size.
 
 ## Requirements
-- [Node](https://nodejs.org) 12.0 or newer
+- [Node](https://nodejs.org) 14.0 or newer
```

Comment:
nit: i think it can work in lower than this node

Reply:
Agreed.  It will probably work on >= 12 because that's the defined engine range in the version of `parcel` being used.  I'll revert this.